includes/payment-methods/class-wc-stripe-express-checkout-ajax-handler.php

```diff
@@ -92,7 +95,7 @@ public function ajax_add_to_cart() {
 		// First empty the cart to prevent wrong calculation.
 		WC()->cart->empty_cart();
 
-		if ( ( 'variable' === $product_type || 'variable-subscription' === $product_type ) && isset( $_POST['attributes'] ) ) {
+		if ( ( ProductType::VARIABLE === $product_type || 'variable-subscription' === $product_type ) && isset( $_POST['attributes'] ) ) {
```

Comment:
Very much non-blocking personal preference (which may be worth discussing as a team): I prefer using fully-qualified references inline as much as possible, mostly because it's not always easy to understand what the class is when viewing isolated sections of the code, especially via diffs.
```suggestion
		if ( ( Automattic\WooCommerce\Enums\ProductType::VARIABLE === $product_type || 'variable-subscription' === $product_type ) && isset( $_POST['attributes'] ) ) {
```

Reply:
Gotcha! To add to this, I found a page with a nice discussion about it: https://www.sitepoint.com/community/t/using-namespace-prefix-versus-use-statement/351357/11